src/web_ui/__init__.py

```diff
@@ -1,27 +1,38 @@
 # Set up logging
 import logging
 
-from multiprocessing.sharedctypes import Value
+from constants import NAME_KEY_MAP
 from flask import Flask, render_template, request
-
 from node_mgmt import NetworkEnv, EnvScanner, get_ssh_fingerprint, Node
 from node_mgmt.commands import (
     imt_sensor_address,
     holykell_sensor_address_7,
     holykell_sensor_address_8,
     trigger_config_generation
 )
+
+
 import os
+import json
+import datetime
+
 from urllib.request import urlopen
 from kvstore import keys, KVStore
+from reader.mqtt_reader import Reader
```

Comment:
This module wasn't necessarily intended for this purpose (it was meant for collecting data from other MQTT brokers), but if it does the job then there's no reason not to use it :)